src/Raven.Embedded/ServerOptions.cs

```diff
@@ -92,7 +94,18 @@ public class SecurityOptions
             public string CertificateLoadExecArguments { get; internal set; }
             public string ServerCertificateThumbprint { get; internal set; }
         }
-    }
 
+        public LicenseOptions LicenseConfiguration { get; set; } = new();
 
+        public class LicenseOptions
+        {
+            public string License { get; set; }
+            public string LicensePath { get; set; }
+            public bool EulaAccepted { get; set; }
+            public bool DisableAutoUpdate { get; set; }
+            public bool DisableAutoUpdateFromApi { get; set; }
```

Comment:
maybe this in 5.4 can be true as well?

Reply:
I think it's better to leave the default value for `DisableAutoUpdateFromApi` property unchanged for both version 5.4 and 6.0. I would prefer not to break the license update logic during the server version upgrade process.